lcm_tools/mock/project.ini

```diff
@@ -0,0 +1,4 @@
+[build]
```

Comment:
IMO we should avoid merging in the mock stuff

Reply:
agreed